### PR TITLE
Cps/fix oscillating cylinder fsi example

### PR DESCRIPTION
**Pull request by LHoeffner**
updated the JSON (for the fluid solver) to fix the oscillating cylinder FSI example

### CODE REVIEW

**Comment by LHoeffner** [comment]
@philbucher where the results diverging also in the original version of the test? (I had to increase the stiffness significantly to ensure convergence. Otherwise the fluid solver stops converging after oscillations induce large mesh distortion. With the original stiffness that occurred after 1.85 sec.)

File: applications/EmpireApplication/test_examples/CoSim_DevExamples/oscilating_cylinder_fsi/ProjectParametersSDoF.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "system_parameters":{
         "mass": 35.75e-3,
-        "stiffness" : 69.48,
+        "stiffness" : 10069.48,
```
Comment (philbucher) [inline applications/EmpireApplication/test_examples/CoSim_DevExamples/oscilating_cylinder_fsi/ProjectParametersSDoF.json]: @mpentek

File: applications/EmpireApplication/test_examples/CoSim_DevExamples/oscilating_cylinder_fsi/ProjectParametersCFD.json [hunk 2]
```diff
@@ -1,173 +1,233 @@
...
-        "domain_size"     : 2,
-        "parallel_type"   : "OpenMP",
-        "echo_level"      : 0,
-        "start_time"      : 0.0,
-        "end_time"        : 10
+    "problem_data": {
+        "problem_name": "CFD_Cylinder",
+        "parallel_type": "OpenMP",
+        "echo_level": 0,
+        "start_time": 0,
```
Comment (philbucher) [inline applications/EmpireApplication/test_examples/CoSim_DevExamples/oscilating_cylinder_fsi/ProjectParametersCFD.json]: why do you change this? This is supposed to be a double

**Review by philbucher — COMMENTED** [review]
@LHoeffner I am not sure if this example is working, we only used it during the development => generally be careful with the things you find in the `test_examples`, they might not always be working, this applies to most apps
=> there is also a proper test for the sdof system in the `tests` folder

@mpentek could you please take a look?

File: applications/EmpireApplication/test_examples/CoSim_DevExamples/oscilating_cylinder_fsi/ProjectParametersCFD.json (hunk 2 above)
Comment (LHoeffner) [reply applications/EmpireApplication/test_examples/CoSim_DevExamples/oscilating_cylinder_fsi/ProjectParametersCFD.json]: thanks for pointing out, I used a Json editor tool to guarantee correct structure of the file but it seems to truncate unnecessary decimals - I restored it in the last commit

**Comment by AndreasWinterstein** [comment]
@philbucher and @LHoeffner  You can also add me, since I will test the sdof and mdof anyway this week. I will also need it.

**Comment by LHoeffner** [comment]
@mpentek and @AndreasWinterstein I guess this is not a top priority request but maybe you can have a look so we can merge it

**Comment by inigolcanalejo** [comment]
@LHoeffner  @philbucher @AndreasWinterstein @mpentek Is this ready to be merged?

**Comment by philbucher** [comment]
@LHoeffner can you please come by our office?

**Comment by LHoeffner** [comment]
> @LHoeffner @philbucher @AndreasWinterstein @mpentek Is this ready to be merged?

yes, for my part

File: applications/EmpireApplication/test_examples/CoSim_DevExamples/oscilating_cylinder_fsi/ProjectParametersCFD.json [hunk 3]
```diff
@@ -1,173 +1,210 @@
...
+    "problem_name": "CFD_Cylinder",
+    "parallel_type": "OpenMP"
+  },
+  "solver_settings": {
+    "solver_type": "ale_fluid",
+    "ale_boundary_parts": [
+      "NoSlip2D_Interface"
+    ],
+    "mesh_motion_solver_settings": {
+      "solver_type": "mesh_solver_structural_similarity"
```
Comment (philbucher) [inline applications/EmpireApplication/test_examples/CoSim_DevExamples/oscilating_cylinder_fsi/ProjectParametersCFD.json]:
```suggestion
      "solver_type": "structural_similarity"
```

**Review by philbucher — APPROVED** [review]

**Pull request merged**